kpi/tests/test_access_logs_export_task.py

```diff
@@ -0,0 +1,124 @@
+import csv
+import os
+
+from django.conf import settings
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+
+from kobo.apps.audit_log.models import AccessLog
+from kpi.models.import_export_task import AccessLogExportTask
+
+User = get_user_model()
```

Comment:
nit (non-blocking)
If you want to avoid this boilerplate, you can use `from kobo.apps.kobo_auth.shortcuts import User`